changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 15 June 2021 12:00 UTC
+
+* [Channel](channels.md#channels) list extended. The latest Channels added have codes `854`, `855`, `856`, `857`, `858`, `859`, `860`, `861`, `862`, `863`, `864`, `864`, `865`, `866`, `867`, `868`, `869`.
```

Comment:
We write there only latest code

Reply:
@AylinKlinge ping

Reply:
@patrikmiksovsky95 what do you mean? Ignore all other new codes and mention only one?

Reply:
@AylinKlinge yes exactly